frontend/src/components/Nav/Search.tsx

```diff
@@ -149,33 +133,31 @@ export default function Search({ className }: { className?: string }) {
             onClick={() => setIsFocused(true)}
         />
         {isLoading && isFocused && <LoadingSpinner className={styles.loadingIcon} />}
-        {isMounted && renderLayer(
-            <AnimatePresence>
-                {isOpen && <motion.ul
-                    {...getMenuProps(layerProps)}
-                    className={classNames(verticalMenuStyles.menu, styles.results)}
-                    style={{
-                        width: triggerBounds.width,
-                        ...layerProps.style,
-                    }}
-                    initial={{ opacity: 0, scaleY: 0.75 }}
-                    animate={{ opacity: 1, scaleY: 1 }}
-                    exit={{ opacity: 0, scaleY: 0.75 }}
-                    transition={{ duration: 0.125 }}
-                >
-                    {items.map((scratch, index) => {
-                        const props = getItemProps({ item: scratch, index })
-                        const oldOnClick = props.onClick
-                        props.onClick = evt => {
-                            evt.preventDefault() // Don't visit the link
-                            return oldOnClick(evt)
-                        }
-
-                        return <a
-                            key={scratch.url}
+        {renderLayer(
+            <ul
+                {...getMenuProps(layerProps)}
+                className={classNames(verticalMenuStyles.menu, styles.results)}
+                style={{
+                    width: (triggerBounds ? triggerBounds.width : "unset"),
+                    ...layerProps.style,
+                }}
+                data-open={isOpen ? "true" : "false"}
```

Comment:
I'd rather use a class here than a data property. You can pass `{ [styles.open]: isOpen }` as an arg to the `classNames` call above to achieve this